xbmc/dialogs/GUIDialogSeekBar.cpp

```diff
@@ -10,8 +10,10 @@
 
 #include "GUIInfoManager.h"
 #include "SeekHandler.h"
-#include "application/Application.h"
```

Comment:
Stupid idea to delete this include here. §ithout it the compiler whines that std::lrintf does not exist (GBM build for raspberry pi 4)

Reply:
uhm, we're not pulling in Application.h for something that sits in <cmath> but of course i'll pull in cmath. thanks.